developers/weaviate/client-libraries/typescript/v2_v3_migration.md

```diff
@@ -8,62 +8,34 @@ image: og/docs/client-libraries.jpg
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import FilteredTextBlock from '@site/src/components/Documentation/FilteredTextBlock';
-import PythonCode from '!!raw-loader!/_includes/code/client-libraries/python_v4.py';
 
 
 :::note Typescript client version
 The current Typescript client version is `v||site.typescript_client_version||`
 :::
 
-The `v3` Weaviate typescript client API is very different to the `v2` API. This guide will help you understand the major changes and how to migrate your code at a high level.
+The `v3` Weaviate Typescript client API is very different to the `v2` API. This guide will help you understand the major changes and how to migrate your code at a high level. 
```

Comment:
Typo:
different from the

Consider:
This guide highlights the major changes in the new client. The guide also makes suggestions about how to migrate your v2 code to v3.